### ISSUE TITLE
Tear out to new window fails

### ISSUE THREAD

**Issue by ColinEberhardt** [ScottLogic/bitflux-openfin#423]
Using the version I started using this morning ...

 - [ ] add details regarding clearing the cache to the README

**Comment by owennw** [comment]
Which version is that?

**Comment by ColinEberhardt** [comment]
Good question, how can I tell?

**Comment by owennw** [comment]
If you're running the app from the exe then it will be the one we pushed to gh-pages. If you're running in dev mode it depends which branch..

**Comment by ColinEberhardt** [comment]
When running from the 'dev' branch I still occasionally see this when I drag to tear out:

![image](https://cloud.githubusercontent.com/assets/1098110/13319327/de4a16d6-dbb8-11e5-8fe4-57ebe871411a.png)

**Comment by owennw** [comment]
Do you have any steps to recreate, or setup required to reproduce that?

**Comment by ColinEberhardt** [comment]
Steps seem to be ..

1. Check out dev branch
2. Run
3. Tear out tile
4. FAIL :-(

i.e. it is happening every time right now!

**Comment by owennw** [comment]
That's very odd. Do you get any error messages? Also have you deleted your local OF cache since you ran older versions of the app? The local storage object shapes have changed between versions and perhaps there's some broken data there?

**Comment by WPFerg** [comment]
Also, what `grunt` command are you running?

**Comment by ColinEberhardt** [comment]
grunt serve

**Comment by WPFerg** [comment]
Try `grunt serve -target http://localhost:5000`

**Comment by ColinEberhardt** [comment]
Stack trace:

```
TypeError: Cannot set property '_owner' of undefined
    at addDesktopEventCallback (<anonymous>:1253:25)
    at Object.fin.desktop.Window.addEventListener (<anonymous>:3379:9)
    at ToolbarCtrl.onReady (app.js:1893)
    at app.js:1863
    at whenReady (<anonymous>:2543:9)
    at Object.fin.desktop.main (<anonymous>:2601:5)
    at Object.ready (app.js:274)
    at new ToolbarCtrl (app.js:1861)
    at Object.invoke (angular.min.js:41)
    at T.instance (angular.min.js:86)(anonymous function) @ angular.min.js:114
```

**Comment by owennw** [comment]
Closed by #444
